### PR TITLE
Project Cleanup

### PR DESCRIPTION
**Pull request by samisuteria**
- Updates to Xcode 11
- Switches to Stable Cocoapods (1.7.5)
- Updates SwiftLint/Snapkit
- Syncs File Structure to Project Structure and removes non-directory groups
- Adds Test Target w/ Dummy Test
- Fixes all project warnings

### CODE REVIEW

**Comment by compnerd** [comment]
Can we split this over a few separate PRs please?  The pods change, the Xcode upgrade, and the tests.  I think that way, if the pods change is merged, I can make progress on the CI setup.  The tests are useful, let’s get that done too.  Can we hold off on the Xcode upgrade until the CI stuff works with that too?

**Comment by samisuteria** [comment]
Even after this PR lands - the project is still compatible with Xcode 10.3 so it should work fine on CI.  Only 1 test was added and its just there to make sure that something runs when you test the app (cmd+u). The pod updates happened while the project was being cleaned up (since there was renaming of targets and project files)

File: Steps4ImpactTests/Steps4ImpactSpec.swift [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import Quick
+import Nimble
+
+class Steps4ImpactSpec: QuickSpec {
```
Comment (samisuteria) [inline Steps4ImpactTests/Steps4ImpactSpec.swift:4]: this is the only test that was added

**Comment by samisuteria** [comment]
Also I'd prefer to get this PR merged in together since all the changes are interdependent on each other. Project/Targets/Dependencies all edit the .xcodeproj file

**Comment by compnerd** [comment]
Okay, since the previous pods update is merged, Id be fine with even just splitting out the Xcode bit into a follow up once we have the CI working (so that I can ensure that I can get that to work properly too)

**Pull request merged**